zbus/src/fdo.rs

```diff
@@ -823,11 +821,11 @@ assert_impl_all!(DBusProxy<'_>: Send, Sync, Unpin);
 
 /// Errors from <https://gitlab.freedesktop.org/dbus/dbus/-/blob/master/dbus/dbus-protocol.h>
 #[derive(Clone, Debug, DBusError, PartialEq)]
-#[dbus_error(prefix = "org.freedesktop.DBus.Error", impl_display = true)]
+#[zbus(prefix = "org.freedesktop.DBus.Error", impl_display = true)]
 #[allow(clippy::upper_case_acronyms)]
 pub enum Error {
     /// Unknown or fall-through ZBus error.
-    #[dbus_error(zbus_error)]
+    #[zbus(zbus)]
```

Comment:
what happened here? :open_mouth: The logical choice for `dbus_error` renaming would be `error` for the macro itself and `dbus_error(zbus_error)` should become `zbus(zbus_error)` (actually come to think of it, perhaps `zbus(fallthrough)` might be even better). I'll fix this as part of my rebase. The rebasing will taking much longer than expected beause of this and some other unrelated and even some unwanted changes I am seeing.

Reply:
ah, nm the point about the macro name. I just keep forgetting the macro is `DBusError` since it's a derive macro unlike proxy and interface.